### PR TITLE
{lang}[GCCcore/8.2.0] Perl v5.28.1

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

I added CPANMinus.
Updated using a small script against the json api @ metacpan. A handful of them I had to fix manually, as well as having to fix some other minor mistake (because CPAN is horribly inconsistent).

### CODE REVIEW

File: easybuild/easyconfigs/p/Perl/Perl-5.28.1-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,1475 @@
...
+        'source_tmpl': 'Parse-RecDescent-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/J/JT/JTBRAUN'],
+        'checksums': ['1943336a4cb54f1788a733f0827c0c55db4310d5eae15e542639c9dd85656e37'],
+    }),
+    ('Carp', '1.50', {
+        'source_tmpl': 'Carp-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/X/XS/XSAWYERX'],
+        'checksums': ['f5273b4e1a6d51b22996c48cb3a3cbc72fd456c4038f5c20b127e2d4bcbcebd9'],
+    }),
+    ('XML::Parser', '2.44', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/Perl/Perl-5.28.1-GCCcore-8.2.0.eb]: @Micket Unintended downgrade? Latest is `2.44_01`, like we have in existing `Perl` easyconfigs, see also `XML-Parser-*.tar.gz` at https://cpan.metacpan.org/authors/id/T/TO/TODDR/

File: easybuild/easyconfigs/p/Perl/Perl-5.28.1-GCCcore-8.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,1475 @@
...
+        'checksums': ['9469a9f398f3a2bf2b0500566ee41d3ff6fae460412a137185767a1cc4783a6d'],
+    }),
+    ('Tree::DAG_Node', '1.31', {
+        'source_tmpl': 'Tree-DAG_Node-%(version)s.tgz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/R/RS/RSAVAGE'],
+        'checksums': ['1c8ba69772568b3758054247097512c550efe31517c329fb65eef7afccc9d304'],
+    }),
+    ('Template', '2.28', {
+        'source_tmpl': 'Template-Toolkit-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/A/AB/ABW'],
```
Comment (boegel) [inline easybuild/easyconfigs/p/Perl/Perl-5.28.1-GCCcore-8.2.0.eb]: This needs to be fixed, correct URL is https://cpan.metacpan.org/authors/id/A/AT/ATOOMIC/Template-Toolkit-2.28.tar.gz, cfr. https://metacpan.org/pod/Template

File: easybuild/easyconfigs/p/Perl/Perl-5.28.1-GCCcore-8.2.0.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/p/Perl/Perl-5.28.1-GCCcore-8.2.0.eb]: That was indeed unintentional. I blame metacpan:
https://fastapi.metacpan.org/v1/release/XML-Parser
Doesn't show 2.44_01. :-1:

Comment (boegel) [reply easybuild/easyconfigs/p/Perl/Perl-5.28.1-GCCcore-8.2.0.eb]: Must be the funky versioning...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2002.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ad62065a1a3852a9a34bd4b1a265b727 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2573.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/5084204284b41915eae6b9c512068f5b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/4039b5ab9155bf3f376a53973b138b83 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Comment by boegel** [comment]
Hmm, it's weird that this is being marked a "Closed" in the header, while it's actually really merged (see https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Perl/Perl-5.28.1-GCCcore-8.2.0.eb)

**Comment by akesandgren** [comment]
I see a "Merged" marker in the header.

**Pull request merged**